cmd/minikube/cmd/config/util.go

```diff
@@ -155,6 +166,10 @@ func enableOrDisableAddonInternal(addon *assets.Addon, cmd bootstrapper.CommandR
 			}
 		}
 	} else {
+		if !ok {
+			console.ErrStyle(console.Conflict, "addon %s is already disabled.", addon.GetAddon())
```

Comment:
same for this too.

Reply:
Address done. Thanks
Append my new UT result:
![image](https://user-images.githubusercontent.com/12505594/59002746-084f1700-8846-11e9-8615-459e1a915f0f.png)